src/storm/storage/sparse/StateValuations.cpp

```diff
@@ -164,6 +164,34 @@ storm::RationalNumber const& StateValuations::getRationalValue(storm::storage::s
     return valuation.rationalValues[variableToIndexMap.at(rationalVariable)];
 }
 
+storm::storage::BitVector StateValuations::getBooleanValues(storm::expressions::Variable const& booleanVariable) const {
+    storm::storage::BitVector result(getNumberOfStates(), false);
+    for (uint64_t stateIndex = 0; stateIndex < getNumberOfStates(); ++stateIndex) {
+        if (getBooleanValue(stateIndex, booleanVariable)) {
```

Comment:
In this case, I would suggest to directly iterate over the valuations instead of using the convenience function `getBooleanValue` which comes with some overhead. For example, we can look up the index of `booleanVariable` once, instead of doing it for each state.

Reply:
You're absolutely right. Thanks!